.circleci/config.yml

```diff
@@ -9,6 +9,9 @@ jobs:
       - run:
           name: Version information
           command: rustc --version; cargo --version; rustup --version
+      - run:
+          name: Install uniffi
+          command: git clone https://github.com/mozilla/uniffi-rs.git; cargo install --path uniffi-rs/uniffi_bindgen
```

Comment:
A placeholder till https://github.com/mozilla/uniffi-rs/issues/253 is resolved.

It can then be replaced with 

```
cargo install uniffi_bindgen
```